cmd/assetsvc/handler.go

```diff
@@ -109,6 +109,21 @@ func extractDecodedNamespaceAndRepoAndVersionParams(params Params) (string, stri
 	return namespace, repo, version, "", nil
 }
 
+func extractDecodedNamespaceAndRepoAndVersionAndCustomFilesParams(params Params) (string, string, string, string, string, error) {
```

Comment:
Probably return an struct is more readable instead to return 4 elements of the same type WDYT ?